core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -218,7 +220,7 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("consumer_prop")
       .ofType(classOf[String])
-    val consumerConfigOpt = parser.accepts("consumer.config", s"Consumer config properties file. Note that $consumerPropertyOpt takes precedence over this config.")
+    val consumerConfigOpt = parser.acceptsAll(Arrays.asList("consumer.config", "command-config"), s"Consumer config properties file. Note that $consumerPropertyOpt takes precedence over this config.")
```

Comment:
This is a related quality of life thing - right now there is inconsistency in flags between commands. Specifically:
* kafka-console-producer uses `producer.config`
* kafka-console-consumer uses `consumer.config`
* most others use `command-config`

I'm trying to _slightly_ simplify the user experience by adding support for `command-config` across the various commands (without changing existing behavior for existing users).

I understand conceptually why we have the `producer.` prefix (since it maps to the corresponding properties in standard Java configurations) but it's somewhat unintuitive for new users to require different flags for similar commands.